inc/saiport.h

```diff
@@ -1813,6 +1946,102 @@ typedef enum _sai_port_attr_t
      */
     SAI_PORT_ATTR_LOOPBACK_MODE,
 
+    /**
+     * @brief MDIX mode status for the port
+     *
+     * @type sai_port_mdix_mode_status_t
+     * @flags READ_ONLY
+     */
+    SAI_PORT_ATTR_MDIX_MODE_STATUS,
+
+    /**
+     * @brief MDIX mode configuration for the port
+     *
+     * @type sai_port_mdix_mode_config_t
+     * @flags CREATE_AND_SET
+     * @default SAI_PORT_MDIX_MODE_CONFIG_AUTO
+     */
+    SAI_PORT_ATTR_MDIX_MODE_CONFIG,
+
+    /**
+     * @brief Configure auto negotiation configuration mode for the port
+     *
+     * @type sai_port_auto_neg_config_mode_t
+     * @flags CREATE_AND_SET
+     * @default SAI_PORT_AUTO_NEG_CONFIG_MODE_DISABLED
+     * @validonly SAI_PORT_ATTR_AUTO_NEG_MODE == true
+     */
+    SAI_PORT_ATTR_AUTO_NEG_CONFIG_MODE,
+
+    /**
+     * @brief Enable auto detection between 1000X and SGMII slave mode
+     *
+     * @type bool
+     * @flags CREATE_AND_SET
+     * @default false
+     * @validonly SAI_PORT_ATTR_MEDIA_TYPE == SAI_PORT_MEDIA_TYPE_FIBER and SAI_PORT_ATTR_SPEED == 1000
+     */
+    SAI_PORT_ATTR_1000X_SGMII_SLAVE_AUTODETECT,
+
+    /**
+     * @brief Enable EEE variable latency mode
+     *
+     * False: Fixed latency
+     * True: Variable latency
+     *
+     * @type bool
+     * @flags CREATE_AND_SET
+     * @default false
+     */
+    SAI_PORT_ATTR_EEE_LATENCY_VARIABLE,
+
+    /**
+     * @brief Configure Fiber module type
+     *
+     * @type sai_port_module_type_t
+     * @flags CREATE_AND_SET
+     * @default SAI_PORT_MODULE_TYPE_1000BASE_X
+     * @validonly SAI_PORT_ATTR_MEDIA_TYPE == SAI_PORT_MEDIA_TYPE_FIBER and SAI_PORT_ATTR_SPEED == 1000
+     */
+    SAI_PORT_ATTR_MODULE_TYPE,
+
+    /**
+     * @brief Configure media types for dual media supported PHY
+     *
+     * @type sai_port_dual_media_t
+     * @flags CREATE_AND_SET
+     * @default SAI_PORT_DUAL_MEDIA_NONE
+     */
+    SAI_PORT_ATTR_DUAL_MEDIA,
+
+    /**
+     * @brief Read cable pair state.
+     * Returns pair states sequentially from list index 0 to n (n = number of pairs - 1)
+     *
+     * @type sai_s32_list_t sai_port_cable_pair_state_t
+     * @flags READ_ONLY
+     */
+    SAI_PORT_ATTR_CABLE_PAIR_STATE,
```

Comment:
is it true, this attribute is only applicable to 1000Base-T? If yes, can you add comment?